### PR TITLE
Convert class component to functional component

### PR DESCRIPTION
**Pull request by rohitksp**

### CODE REVIEW

File: rohit/Image-Search-Hooks/src/component/SearchBar.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+class SearchBar extends React.Component {
```
Comment (nrchandan) [inline rohit/Image-Search-Hooks/src/component/SearchBar.js]: Let's convert this into a functional component as well?

File: rohit/Image-Search-Hooks/src/component/ImageCard.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+class ImageCard extends React.Component {
```
Comment (nrchandan) [inline rohit/Image-Search-Hooks/src/component/ImageCard.js]: Let's convert this into functional as well? Why waste opportunity to practice some more!

**Review by nrchandan — APPROVED** [review]
Looks good, but you could convert two more components to functional as well.

File: rohit/Image-Search-Hooks/src/component/SearchBar.js (hunk 1 above)
Comment (rohitksp) [reply rohit/Image-Search-Hooks/src/component/SearchBar.js]: okay we will do.

**Pull request merged**